README.md

```diff
@@ -10,83 +10,67 @@ Algolia Search for Magento 2
 
 -------
 
-🔎  &nbsp; **Need help?** Check out our [Technical Troubleshooting Guide](https://www.algolia.com/doc/integration/magento-2/troubleshooting/technical-troubleshooting/). For feedback, bug reporting, or unresolved issues with the extension, please contact us at [support@algolia.com](mailto:support@algolia.com). Please include your Magento version, extension version, application ID, and steps to reproducing your issue. Add additional information like screenshots, screencasts, and error messages to help our team better troubleshoot your issues.
+## Features
 
--------
-
-#### Magento 2.4 compatibility & extension's versions End of Life
-
-We are happy to announce that the version 3.x of our extension is now compatible with Magento 2.4. Review the [Customisation](https://github.com/algolia/algoliasearch-magento-2#customisation) section to learn more about the differences between our extension versions.
-
-Additionally, we are announcing the end of life for our legacy versions. We will continue to support and backport major changes to the minor branches until the defined dates below. We will not accept community PRs for those branches after this date. 
-
-| Extension Version | End of Life |
-| --- | --- |
-| v1.x | Dec 2020 |
-| v2.x | Dec 2020 |
-| v3.x | N/A |
+The Algolia extension replaces the default search in Magento Open Source and Adobe Commerce with a robust autocomplete search menu and instant search results page. with an autocomplete search menu and an instant search results page that updates with products and categories in real time.
```

Comment:
Remove with an autocomplete search menu and an instant search results page that updates with products and categories in real time.